docs/events/timeline-templates.asciidoc

```diff
@@ -103,41 +104,43 @@ value is retrieved from the alert's `process.name` field.
 [[man-templates-ui]]
 === Manage existing Timeline templates
 
-You can view, duplicate, delete, and create templates from existing Timelines:
+You can view, duplicate, export, delete, and create templates from existing Timelines:
 
-. Go to *Timelines* -> *Templates*.
+. Go to *Investigate* -> *Timelines* -> *Templates*.
 
 +
 [role="screenshot"]
 image::images/all-actions-timeline-ui.png[]
 
 . Click the *All actions* icon in the relevant row, and then select the action:
 
-* *Create timeline from template*
+* *Create timeline from template* (see <<create-timeline-template>>)
 * *Duplicate template*
-* *Delete template*
+* *Export selected* (see <<import-export-timeline-templates>>)
+* *Delete selected*
 
 TIP: To perform the same action on multiple templates, select templates and
 then the required action from the *Bulk actions* menu.
```

Comment:
TIP: Let's try not to use `"and then"` because it's wordy. In some instances it's OK, but in general, let's try to avoid it. 

```suggestion
TIP: To perform the same action on multiple templates, select the appropriate templates,
then the required action from the *Bulk actions* menu.
```